Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs

```diff
@@ -1712,11 +1709,14 @@ private static string GetPostProcessArguments(string path, string arguments)
 
         private void Process_Exited(object sender, EventArgs e)
         {
-            using (var process = (IProcess)sender)
+            try
```

Comment:
Why replace the using statement with a try-finally?

Reply:
Hmm I'm not really sure why I did this 😅

I guess I'm used to seeing `using` only used when the code is actually creating the object at the same time (i.e. `using (var process = new Process())`). Is this a real convention or just one that lives in my head?

Reply:
No idea, but `using` is just syntax sugar for `try-finally`

Reply:
Fixed.